nearcore/src/shard_tracker.rs

```diff
@@ -99,6 +94,7 @@ impl ShardTracker {
         shard_id: ShardId,
         is_me: bool,
     ) -> bool {
+        // TODO: fix these unwrap_or here and handle error correctly. The current behavior masks potential errors and bugs
```

Comment:
Do we have an issue for this?

Reply:
https://github.com/near/nearcore/issues/4936